babelsberg/constraintinterpreter.js

```diff
@@ -509,7 +508,7 @@ Object.subclass('Constraint', {
     addConstraintVariable: function(v) {
         if (v && !this.constraintvariables.include(v)) {
             this.constraintvariables.push(v);
-            v.logTimings = this.logTimings;
+            v.logTimings = this._options.logTimings;
```

Comment:
in that branch you simply removed this line because you have introduced the `definingConstraint`, so yeah... I guess fixing the logging output (especially the timing stuff) should be done after the feature branches are merged....